sdk/core/Azure.Core.Experimental/api/Azure.Core.Experimental.netstandard2.0.cs

```diff
@@ -86,6 +95,13 @@ public partial class JsonData : System.Dynamic.IDynamicMetaObjectProvider, Syste
         public System.Threading.Tasks.Task<long> WriteToAsync(System.IO.Stream stream, System.Threading.CancellationToken cancellationToken) { throw null; }
     }
 }
+namespace Azure.Core.Experimental
```

Comment:
Can we move this into `Azure` so we don't have to add a namespace?

Reply:
Sure

Reply:
Moved to Azure.Core.Pipeline